lib/driver.js

```diff
@@ -332,6 +324,14 @@ class IosDriver extends BaseDriver {
       return;
     }
 
+      // if we have an ipa file, set it in opts
+      if (this.opts.app) {
```

Comment:
I almost want to say that if we are in this method and _don't_ have an ipa at this point, that is a problem, no? Given that we can only install ipas to real devices?

I wonder if the whole distinction between `this.opts.app` and `this.opts.ipa` even makes sense anymore?